pkg/kubelet/cm/topologymanager/policy_options.go

```diff
@@ -27,12 +27,14 @@ import (
 
 const (
 	PreferClosestNUMANodes string = "prefer-closest-numa-nodes"
+	MaxAllowableNUMANodes  string = "max-allowable-numa-nodes"
 )
 
 var (
 	alphaOptions = sets.New[string]()
 	betaOptions  = sets.New[string](
 		PreferClosestNUMANodes,
+		MaxAllowableNUMANodes,
```

Comment:
The new option should start as `alpha` option and graduate independently.

Reply:
Thanks! Done.

Reply:
Yes, this needs to change. The new option should be put directly into beta.

Reply:
it seems this is still in the `alpha` options in the last upload?